docs/release-notes/release-notes.md

```diff
@@ -174,99 +172,99 @@ To check for security updates, go to [Security announcements for the Elastic sta
 * Add knex@3 instrumentation. ([#3659](https://github.com/elastic/apm-agent-nodejs/pull/3659))
 * Update [*OpenTelemetry bridge*](/reference/opentelemetry-bridge.md) support to `@opentelemetry/api` version 1.7.0.
 
-### Fixes [4-2-0-fixes]
+### Fixes [elastic-apm-nodejs-agent-420-fixes]
 * Fix `mongodb` instrumentation to avoid loosing context when multiple cursors are running concurrently. ([#3161](https://github.com/elastic/apm-agent-nodejs/issues/3161))
 * Set `mongodb` span’s outcome according to the result of the command being traced. ([#3695](https://github.com/elastic/apm-agent-nodejs/pull/3695))
 * Fix `@aws-sdk/client-sqs` instrumentation which was failing for `SendMessageBatch` command when any of the entities does not contain `MessageAttributes`. ([#3746](https://github.com/elastic/apm-agent-nodejs/issues/3746))
 
-## 4.1.0 [4-1-0]
+## 4.1.0 [elastic-apm-nodejs-agent-410-release-notes]
 **Release date:** October 9, 2023
 
-### Features and enhancements [4-1-0-features-enhancements]
+### Features and enhancements [elastic-apm-nodejs-agent-410-features-enhancements]
 * Update [*OpenTelemetry bridge*](/reference/opentelemetry-bridge.md) support to `@opentelemetry/api` version 1.6.0. [#3622](https://github.com/elastic/apm-agent-nodejs/pull/3622)
 * Add support for `@aws-sdk/client-dynamodb`, one of the AWS SDK v3 clients. ([#2958](https://github.com/elastic/apm-agent-nodejs/issues/2958))
 * Add support for `@aws-sdk/client-sns`, one of the AWS SDK v3 clients. ([#2956](https://github.com/elastic/apm-agent-nodejs/issues/2956))
 * Add support for `@aws-sdk/client-sqs`, one of the AWS SDK v3 clients. ([#2957](https://github.com/elastic/apm-agent-nodejs/issues/2957))
 * Fixes for some values of the [`disableInstrumentations`](/reference/configuration.md#disable-instrumentations) config setting. "redis" will now properly disable instrumentation for redis@4. "next" will propertly disable all Next.js instrumentation. ([#3658](https://github.com/elastic/apm-agent-nodejs/pull/3658))
 
-### Fixes [4-1-0-fixes]
+### Fixes [elastic-apm-nodejs-agent-410-fixes]
 * Changes to cloud metadata collection for Google Cloud (GCP). Most notably the `cloud.project.id` field is now the `project-id` from [https://cloud.google.com/compute/docs/metadata/default-metadata-values#project_metadata](https://cloud.google.com/compute/docs/metadata/default-metadata-values#project_metadata) rather than the `numeric-project-id`. This matches the value produced by Elastic Beats (like filebeat). [#3614](https://github.com/elastic/apm-agent-nodejs/issues/3614)
 
-## 4.0.0 [4-0-0]
+## 4.0.0 [elastic-apm-nodejs-agent-440-release-notes]
```

Comment:
Is this `440` error in the anchor name the reason that the `4.0.0` entry in the "On this page" TOC in the wrong place?

See https://docs-v3-preview.elastic.dev/elastic/apm-agent-nodejs/pull/4524/release-notes/release-notes

![Screenshot 2025-03-19 at 11 01 29 AM](https://github.com/user-attachments/assets/7c1dfb69-6fb4-4931-9e86-976450cda42a)